docs/planning/sprintplanning.md

```diff
@@ -0,0 +1,66 @@
+- All new proposals are discussed either in WG meeting, or on slack, or dedicated meetings if needed at any point scheduled by the proposer. 
+- By R-2w’s WG meeting all new proposals should be vetted and verified that they fit the [Planning criteria](#planning-criteria). Features that do not fit this criteria will not be considered in the current release planning cycle and hence early discussion is encouraged instead of waiting for R-2w.
+- All eligible proposals will then be snapshotted into the voting document and open for voting till R-1w WG meeting. In R-1w WG meeting top features based on votes will be planned for next release. 
+> We need some kind of estimates for each feature for release planning. Shall we adopt story points on fibonacci scale or t-shirt sizing? My vote goes to story points on fibonacci scale to avoid spending a lot of time on estimation, which is a common pitfall.
+- Selected features will be labeled “triage/accepted”. Rest will be labelled “triage/ice-boxed” and will be eligible for next planning cycle. Accepted features will be added to the correct milestone and sprint specific project for tracking.
+
+## How to manage Epic/Scenario?
+- Epic/Scenario typically spans 1-3 months or even more in some cases.
+- For new proposals these will be created and triaged/refined just like Story/Feature. However, after triaging these need to be broken down into Features for planning and voting.
+- Github doesn’t allow nested Issues, hence the only good way to track these are by creating Projects and adding the broken down features as issues to this project. An issue can be part of multiple projects and hence we can get a Scenario\Epic view as well as a Sprint view through respective projects.
+
+## Planning criteria:
+- Features are well defined and scoped with clear exit criteria. No feature creeps once release starts
+- Features can be implemented in <2w by one engineer. In case of Epic/Scenario, it could be broken down into <2w features with clear exit criteria.
+- The engineering team should be confident that any engineer can implement the feature based on the content in the feature *(This is similar to refinement in a scrum, but doesn’t have to be as strict as a functional specification)*.
+
+## Organizing with Github Projects 
+- New Project per sprint. Today, the sprint and milestone are both aligned. However in future we may decide to have multiple sprints in a release cycle.
+- New Project per any Story/Scenario
+- Any other projects that help organize and will be created when needed.
+
+## Organizing with Github Milestone 
+- A new milestone is created in Github for each 6 weeks release cycle.
+- All features and bugs that are planned for the release cycle are assigned to the release for tracking the progress of the milestone.
+
+## Pending work and proposed changes:
+- Decide estimation technique (story points on fibonacci, t-shirt size etc)
+- Create features and bugs for 0.6 release.
```

Comment:
Can we start with just features (since we have a lot of work already, and I'd like to be able to address bugs within a release window)?

Reply:
We already have a list of bugs, existing and from current PRs that need to be triaged. I would suggest lets start tracking both bugs and features.

Let's postpone estimation. We dont need it are current stage

Reply:
+1 for postponing estimation. No clear usage for me right now. However I would love to suggest to have a label in the triage similar to the one in build-pipeline (tekton) called [`meaty-juicy-coding-work`](https://github.com/tektoncd/pipeline/issues?q=is%3Aissue+is%3Aopen+label%3Ameaty-juicy-coding-work  )